packages/crayons-core/src/components/checkbox/checkbox.scss

```diff
@@ -1,10 +1,14 @@
+/**
+  @prop --checkbox-margin-bottom: Bottom Margin of the checkbox.
+*/
+
 :host {
   display: inline-block;
   position: relative;
 }
 
 .checkbox-container {
-  margin-bottom: 4px;
+  margin-bottom: var(--checkbox-margin-bottom, 4px);
```

Comment:
Kishore, we actually don't need margin-bottom of 4px here, have removed it in the PR for form render. Since you are passing as 0px anyway, I think we can remove this. Even in DSM there is no specification of margin-bottom for checkbox.

Reply:
Thanks for the confirmation Shravan, I have removed the margin-bottom here.